introduction_to_amazon_algorithms/forecasting_services_comparison/sagemaker_example.ipynb

```diff
@@ -0,0 +1,1543 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    version = 1
</pre><p>Same comment from the other notebook applies to version number versus timestamp.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3532/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>